src/styles/blocks/pages.scss

```diff
@@ -0,0 +1,6 @@
+.pages {
+  display: flex;
+  flex-direction: column;
+  max-width: 1260px;
+  margin: 0 auto;
```

Comment:
Don't give margins to BEM-blocks
You may use mix of classes like this:
`<main class="main main__content">`
and give this margin to the element `main__content`